types/iva/defects/get/index.d.ts

```diff
@@ -20,6 +18,9 @@ export interface DefectGETIVA {
     additionalInfo: boolean;
     inspectionTypes: InspectionType[];
```

Comment:
is this inspectionTypes array optional? Needs to be as we know some manuals don't have this data